controllers/applicationvolumereplication_controller_test.go

```diff
@@ -269,3 +269,166 @@ var _ = Describe("ApplicationVolumeReplication controller", func() {
 		})
 	})
 })
+
+var _ = Describe("isManifestInAppliedState", func() {
+	timeOld := time.Now().Local()
+	timeMostRecent := timeOld.Add(time.Second)
+
+	Describe("single timestamp", func() {
```

Comment:
Why didn't you add `Context` and have single Describe? Something like:
```
Describe("Reading ManifestWork")
    Context("When Single Timestamp")
        It("'Applied' present".......
        ...
        ...
        ...
    Context("When multiple timestamps")
       It("no duplicates, 'Applied' is most recent")
       ...
```

Reply:
No good reason. It was my first time using Ginkgo and `Describe` and `Context` appear functionally equivalent, so I didn't give it much thought. I think your proposed style is easier to read, so I'll change to that.